library/wizard/update_book.py

```diff
@@ -0,0 +1,30 @@
+# See LICENSE file for full copyright and licensing details.
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+
+
+class UpdateBooks(models.TransientModel):
+    """Defining update books."""
+
+    _name = "update.books"
+    _description = "Update Books"
+
+    name = fields.Many2one(
+        "product.product", "Book Name", required=True, help="Book name"
+    )
+
+    def action_update_books(self):
+        lib_book_obj = self.env["library.book.issue"]
+        for rec in self:
+            if rec.name.availability == "notavailable":
+                raise ValidationError(
+                    _(
+                        """This Book is not available! \
+Please try after sometime !"""
+                    )
+                )
+            if self._context.get("active_ids"):
+                for active_id in self._context.get("active_ids"):
```

Comment:
Directly browse and execute the loop so it will save the execution of method by reducing the line 29 "book_rec = lib_book_obj.browse(active_id)"